content/es/guides/directory-structure/assets.md

```diff
@@ -0,0 +1,286 @@
+---
+title: recursos
+description: El directorio de `assets` (recursos), contiene recursos sin compilar, como imágenes, fuentes de texto o archivos de Sass y Stylus.
+position: 2
+category: directory-structure
+csb_link: https://codesandbox.io/embed/github/nuxt-academy/guides-examples/tree/master/04_directory_structure/02_assets?fontsize=14&hidenavigation=1&theme=dark
+videoScript:
+  - assets-video.md
+questions:
+  - question: Cuál directorio contien los archivos sin compilar, Stylus o Sass, imágenes, o fuentes?
```

Comment:
After the word _directorio_, the word **_cotien_** should be **contiene**. Also I think that the word _Cúal_ can be substituted for the word _Qué_ and _directorio_ to plural (_directorios_), so it reads more naturally, leaving the sentence like this:
> Qué directorios contienen los archivos sin compilar, Stylus o Sass, imágenes, o fuentes?

Reply:
Ohh wow this one is a bad one ahahaha. 

- [ ] Fix